.github/workflows/cut_release_branch.yml

```diff
@@ -0,0 +1,111 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This workflow will update apache beam master branch with next release version
+# and cut release branch for current development version.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Cut Release Branch
+on:
+  workflow_dispatch:
+    inputs:
+      RELEASE_VERSION:
+        description: Beam version of current release
+        required: true
+      NEXT_VERSION:
+        description: Next release version
+        required: true
+
+jobs:
+  update_master:
+    needs: update_release_branch
+    runs-on: [self-hosted, ubuntu-20.04]
+    env:
+      MASTER_BRANCH: master
+      NEXT_RELEASE: ${{ github.event.inputs.NEXT_VERSION }}
+      SCRIPT_DIR: ./release/src/main/scripts
+    steps:
+      - name: Validate Next Version
+        run: |
+          if [[ $NEXT_RELEASE =~ ([0-9]\.[0-9]*\.[0-9]) ]]; then
```

Comment:
The regex should be `[0-9]+\.[0-9]+\.[0-9]+` here and in the error message below.